Documentation/building/cross-building.md

```diff
@@ -103,3 +103,20 @@ prajwal@ubuntu ~/corefx $ ./scripts/arm32_ci_script.sh \
 ```
 
 The Linux ARM Emulator is based on the soft floating point and thus the native binaries are generated for the armel architecture. The corefx binaries generated by the above command can be found at `~/corefx/bin/Linux.armel.Release`, `~/corefx/bin/Linux.AnyCPU.Release`, `~/corefx/bin/Unix.AnyCPU.Release`, and `~/corefx/bin/AnyOS.AnyCPU.Release`.
+
+
+Build corefx for a new architecture
+===================================
+
+When building for a new architecture you will need to build the native pieces separate from the managed pieces in order to correctly boot strap the native runtime. Instead of calling build.sh directly you should instead split the calls like such:
+
+Example building for armel
+```
+build-native.sh -buildArch=armel
+--> Output goes to bin/runtime/netcoreapp-Linux-Debug-armel
+
+build-managed.sh -buildArch=x64
+--> Output goes to bin/runtime/netcoreapp-Linux-Debug-x64
+```
+
+The reason you need to build the managed portion for x64 is to because it depends on runtime packages for the new architecture which don't exist yet so we use another existing architecture such as x64 as a proxy for building the managed binaries. Once all the necessary builds are setup and packages are published this spliting of the build should no longer be needed.
```

Comment:
@weshaggard FYI. You can find above workaround using `netcoreapp-Linux-Debug-x64` at #16378 (link to the code: https://github.com/dotnet/corefx/pull/16378/files#diff-9b74089ede263293591a48bb7430a9a4R254)